.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Checklist:
+ - Write / update unit tests
+ - Write / update integration (envtest) tests
+ - Ensure there is an issue for e2e tests if needed
+ - Make sure `make test` passes
+ - Test code changes manually
```

Comment:
Is this covered by the first two?

Reply:
No, it's not. The first two says about unit tests, where as this item is about deploying changes to dev cluster and try the functionality manually.

Reply:
+1 I'd expect this would be covered by integraton test

Reply:
I would say if the feature isn't covered by e2e tests, it needs a manual test as well. The way I understand it, integration tests using envtest are just an approximation of a real cluster, not to mention that they don't test the interaction with the rest of RHTAP